Files.Launcher/MessageHandlers/FileOperationsHandler.cs

```diff
@@ -810,6 +810,10 @@ public void TryCancel(string uid)
 
             private void UpdateTaskbarProgress()
             {
+                if (OwnerWindow == null || taskbar == null)
```

Comment:
Unrelated to current PR: fixes an issue preventing file rename in the fulltrust process